scripts/wdio/getWdioConfigPath.js

```diff
@@ -1,16 +1,14 @@
 const path = require('path');
-const fs = require('fs');
 const Logger = require('../utils/logger');
-
-const isFile = filePath => (fs.existsSync(filePath) && !fs.lstatSync(filePath).isDirectory());
+const { isFile } = require('../../config/configUtils');
 
 const getWdioConfigPath = (configPath) => {
   if (configPath) {
     return path.resolve(configPath);
   }
 
   // Try to find the local to process.cwd webpack config
-  const localConfig = path.resolve(process.cwd(), 'wdio.conf.js');
+  const localConfig = path.resolve(process.cwd(), './wdio.conf.js');
```

Comment:
was this working previously?

Reply:
idk why I changed this. Reverted: https://github.com/cerner/terra-toolkit/pull/298/commits/9ff9152b60a139e62cc5c4e31afaddf077668ccf#diff-2b14bdc794a09b6c5703f768f8877711R11